modules/common-attributes.adoc

```diff
@@ -53,10 +53,10 @@ endif::[]
 :osp: Red Hat OpenShift
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 4.0.2
+:rhacs-version: 4.1.1
 :ocp-supported-version: 4.10
 :product-title: Red Hat Advanced Cluster Security for Kubernetes
-:product-version: 4.0
+:product-version: 4.1
```

Comment:
Not related to this line. Is a content type needed for this file? It's likely already there.

Reply:
Thanks @nalhadef Right now it doesn't have a content type, however I'll check what OCP docs uses for the common attributes file and change it.